.github/workflows/test.yml

```diff
@@ -11,23 +11,28 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        go-version: [1.17.x, 1.18.x]
-        os: [ubuntu-latest, macos-latest, windows-latest]
+        go-version:
+        - '1.18.x'
+        - '1.19.x'
+        os:
+        - ubuntu-latest
+        - macos-latest
+        - windows-latest
     runs-on: ${{ matrix.os }}
     steps:
+    - name: Checkout code
+      uses: actions/checkout@v3
     - name: Install Go
       uses: actions/setup-go@v3
       with:
         go-version: ${{ matrix.go-version }}
-    - name: Checkout code
-      uses: actions/checkout@v3
     - name: Test
       run: |
         go test ./...
         go test -race ./...
 
     - name: Tidy
-      if: matrix.os == 'ubuntu-latest' # no need to do this everywhere
+      if: matrix.os == 'ubuntu-latest' && matrix.go-version == 'stable' # no need to do this everywhere
```

Comment:
presumably this is wrong as it still mentions "stable"?

Reply:
/facepalm Good catch, thanks.